app/src/main/java/org/systers/mentorship/remote/requests/TaskRequest.kt

```diff
@@ -0,0 +1,9 @@
+package org.systers.mentorship.remote.requests
+
+/**
+ * This data class represents all data necessary to create a task for a relation between two users
+ * @param description represents description of the task
+ */
+data class TaskRequest(
+        val description: String
+)
```

Comment:
This class is not needed for this task. This is for creating a new task, please remove this class